### DISCUSSION TITLE
Help setting background to black

### DISCUSSION THREAD

**Discussion by groovyghoul** [LazyVim/LazyVim#247] [Q&A] [answered]
I was hoping someone might be able to help me out with setting the background to black. I am using the gruvbox recipe from the example file, but I can't figure out where to put the `vim.cmd` commented out in the code bolow.

Running the command `highlight Normal ctermbg=16 guibg=#000000` works perfectly, I just can't figure out where to put it in the config.

Thanks. 

```lua
return {
  -- add gruvbox
  {
    "ellisonleao/gruvbox.nvim",
  },

  -- Configure LazyVim to load gruvbox
  {
    "LazyVim/LazyVim",
    opts = {
      colorscheme = "gruvbox",
    },
    -- config = function()
    --   vim.cmd("highlight Normal ctermbg=16 guibg=#000000")
    -- end,
  },
}
```

**Comment by loiccoyle** [comment, marked as answer]
You can override highlight groups (see the readme https://github.com/ellisonleao/gruvbox.nvim)  something like: 
```lua
return {
    "ellisonleao/gruvbox.nvim",
    opts = {
        overrides = {
            Normal = {bg = "#000000"}
        }
    }
}
```

**Reply by groovyghoul** [reply]
You're the best! Thank you so much!

**Reply by ejsadiarin** [reply]
hey @loiccoyle not related to the question but, where do you find these values (like the "Normal") for the overrides field?

I can't seem to find it in the gruvbox.nvim docs.

**Reply by dpetka2001** [reply]
I don't use the colorscheme, but `Normal` is defined here in gruvbox's [docs](https://github.com/ellisonleao/gruvbox.nvim/blob/477c62493c82684ed510c4f70eaf83802e398898/lua/gruvbox.lua#L288). Also see `:h highlight-groups`.

**Reply by ejsadiarin** [reply]
ohhh thank you so much @dpetka2001 !

**Reply by loiccoyle** [reply]
@ejsadiarin You can also run the `highlight` command to see all the highlight groups and their values.
